txp/Pos/Txp/Toil/Utxo/Util.hs

```diff
@@ -33,3 +35,11 @@ utxoToStakes = foldl' putDistr mempty . M.toList
   where
     plusAt hm (key, val) = HM.insertWith unsafeAddCoin key val hm
     putDistr hm (_, toaux) = foldl' plusAt hm (txOutStake toaux)
+
+utxoToAddressCoinPairs :: Utxo -> [(Address, Coin)]
```

Comment:
Another reason to do it is that it makes clearer to the caller that the returned value doesn't have any duplicated addresses (the function with such name could also return outputs converted to pairs which would have duplicates).